open-source-terms.md

```diff
@@ -0,0 +1,259 @@
+# npm Open-Source Terms
+
+These npm Open Source terms of use (these _Terms_) govern access to
+and use of <https://www.npmjs.com> (the _Website_) as well as the
+"npm Public Registry" at <https://registry.npmjs.com> (the _Public
+Registry_). Both the Website and the Public Registry are operated by
+npm, Inc. (_npm_). The Website and the Public Registry are referred to
+together as _npm Open Source_ throughout these Terms.
+
+These npm Open Source Terms were last updated on November 9, 2015.
```

Comment:
Yeah, let's not do anything that requires additional engineering lift.  @kemitchell what do you think of @anseljh's suggestion?  We typically link to the history for that specific doc, so the url would be: https://github.com/npm/policies/commits/open-source-terms/open-source-terms.md